shell/platform/embedder/embedder_external_view_embedder.cc

```diff
@@ -237,8 +227,15 @@ bool EmbedderExternalViewEmbedder::SubmitFrame(GrContext* context) {
     return false;
   }
 
+  // Copy the contents of the root picture recorder onto the root surface
+  // while making sure to take into account any surface transformations.
   if (auto root_canvas = root_render_target_->GetRenderSurface()->getCanvas()) {
+    root_canvas->setMatrix(pending_surface_transformation_);
+    root_canvas->clear(SK_ColorTRANSPARENT);
```

Comment:
Isn't this redundant for the root canvas? do you know if there's a performance cost for doing it twice?

Reply:
No. The embedder may have unnecessary stuff on the render target that we need to get rid of.